src/functions/Functions.php

```diff
@@ -93,13 +91,6 @@ public function countArguments(): array
      */
     public function countArgumentsWrapper(): array
     {
-        $arg = func_get_args();
-        foreach ($arg as $value) {
-            if (!is_string($value)) {
```

Comment:
why did you delete this code?

Reply:
Original repo has been changed and those lines were removed.